src/main/resources/static/js/testrules.js

```diff
@@ -177,91 +178,115 @@ jQuery(document).ready(
               sticky : false,
               theme : 'Notify'
             });
-            
+
             var list = JSON.parse(fileContent);
-            if (isRules == true) {
-                vm.rulesBindingList([]);
-                list.forEach(function(element) {
-                    vm.addRule(element);
-                });
+            if (isRules) {
+                if (isReplacing) {
+                    vm.rulesBindingList([]);
+                    list.forEach(function(element) {
+                        vm.addRule(element);
+                    });
```

Comment:
duplicated code.
you may do 
if (isReplacing) {
   vm.rulesBindingList([]);
}
only.

Reply:
Done